lib/membrane/srtp_keying_material_event.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Membrane.SRTP.KeyingMaterial.Event do
```

Comment:
The convention is rather to have 

```suggestion
defmodule Membrane.SRTP.KeyingMaterialEvent do
```